new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -0,0 +1,555 @@
+import static oracle.weblogic.kubernetes.actions.TestActions.defaultAppParams;
+import static oracle.weblogic.kubernetes.actions.TestActions.defaultWITParams;
+import static oracle.weblogic.kubernetes.actions.TestActions.deleteDomainCustomResource;
+import static oracle.weblogic.kubernetes.actions.TestActions.deleteImage;
+import static oracle.weblogic.kubernetes.actions.TestActions.deleteNamespace;
+import static oracle.weblogic.kubernetes.actions.TestActions.deleteServiceAccount;
+import static oracle.weblogic.kubernetes.actions.TestActions.dockerLogin;
+import static oracle.weblogic.kubernetes.actions.TestActions.dockerPush;
+import static oracle.weblogic.kubernetes.actions.TestActions.helmList;
+import static oracle.weblogic.kubernetes.actions.TestActions.installOperator;
+import static oracle.weblogic.kubernetes.actions.TestActions.uninstallOperator;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.dockerImageExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.domainExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.operatorIsRunning;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.serviceReady;
+import static oracle.weblogic.kubernetes.utils.FileUtils.checkDirectory;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.awaitility.Awaitility.with;
+
+// Test to install Operator, create model in image domain and verify the domain
+// has started successfully
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test to create model in image domain and start the domain")
+@ExtendWith(Timing.class)
+class ItMiiDomain implements LoggedTest {
+
+  // operator constants
+  private static final String OPERATOR_RELEASE_NAME = "weblogic-operator";
+  private static final String OPERATOR_CHART_DIR =
+      "../kubernetes/charts/weblogic-operator";
+  private static final String OPERATOR_IMAGE =
+      "oracle/weblogic-kubernetes-operator:3.0.0";
+  //"phx.ocir.io/weblogick8s/weblogic-kubernetes-operator:develop";
+
+  // mii constants
+  private static final String WDT_MODEL_FILE = "model1-wls.yaml";
+  private static final String MII_IMAGE_NAME = "mii-image";
+  private static final String APP_NAME = "sample-app";
+
+  // domain constants
+  private static final String DOMAIN_VERSION = "v7";
+  private static final String API_VERSION = "weblogic.oracle/" + DOMAIN_VERSION;
+
+  private static HelmParams opHelmParams = null;
+  private static V1ServiceAccount serviceAccount = null;
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private String domainUID = "domain1";
+  private String miiImage = null;
+
+  /**
+   * Install Operator.
+   */
+  @BeforeAll
```

Comment:
the description above suggests that the installation of the operator is a part of the test - so I think that either you should install the operator in the test itself -or- update the description if the installation is not actually part of the test, but the operator is just a pre-req of th test

Reply:
yes, operator is just a pre-req for the test, will change the description if per-requisites should not be mentioned there.

Reply:
done